### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a step for clarification on security defaults

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/932

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -52,7 +52,8 @@ If you have previously set up MFA with baseline policies, [you must turn them of
 1. Sign in to [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822) with your Global admin credentials.
 2. Go to [Azure Active Directory Properties](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Properties).
 
-2. On the bottom of the page, choose **Manage Security defaults**.
+3. On the bottom of the page, choose **Manage Security defaults**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
3. At the bottom of the page, choose **Manage Security defaults**.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ If you have previously set up MFA with baseline policies, [you must turn them of
...
+4. Choose **Yes** to Enable Security Defaults and **No** to Disable security Defaults.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
4. Choose **Yes** to enable security defaults and **No** to disable security defaults.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 3]
```diff
@@ -76,6 +77,10 @@ All Office 2016 client applications support MFA through the use of the Active Di
 
     ![Modern authentication panel with enable checkbox checked.](../media/enablemodernauth.png)
     
+> [!TIP]
+>  Tell your users how to set up the Authenticator app, please visit [Use Microsoft Authenticator with office 365](https://support.office.com/en-us/article/use-microsoft-authenticator-with-office-365-1412611f-ad8d-43ab-807c-7965e5155411?ui=en-US&rs=en-US&ad=US#ID0EAADAAA=_Step_1).
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
> To explain to your users how to set up the Authenticator app, please visit [Use Microsoft Authenticator with office 365](https://support.office.com/article/use-microsoft-authenticator-with-office-365-1412611f-ad8d-43ab-807c-7965e5155411?ui=en-US&rs=en-US&ad=US#ID0EAADAAA=_Step_1).
```

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@Sirkkuw -  PR has been copy edited and is ready for final review. This PR incorporates your recommendations based on these issues- #932 and #937 

Thank you.

cc: @kenwith

**Pull request merged**